Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,15 +1,11 @@
-﻿using System;
-using BootCamp.Chapter.Logger;
-
-namespace BootCamp.Chapter
+﻿namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
-            BmiCalculator.Logger.Warn("Program started.");
-            BmiCalculator.CalculateBmi();
-            BmiCalculator.Logger.Warn("Program finished.");
+            var bmiInstance = new CalculateBmi();
```

Comment:
Instance is redundant. Use real life words for naming stuff.
It's a calculator. Since the scope is just 2 lines, that is a good enough name to clarify the intention.